Documentation/Books/Manual/Administration/ManagingUsers/README.md

```diff
@@ -1,40 +1,214 @@
 Managing Users
 ==============
 
-The user management in ArangoDB 3 is similar to the ones found in
-MySQL, Postgres, or other database systems.
+The user management in ArangoDB 3 is similar to the ones found in MySQL,
+PostgreSQL, or other database systems.
 
-An ArangoDB server contains a list of users. Each user can have access
-to one or more databases (or none for that matter).
+## Actions and Access Levels
+
+An ArangoDB server contains a list of users. It also defines various
+access levels that can be assigned to a user (for details, see below)
+and that are needed to perform certain actions. These actions can be grouped
+into three categories:
+
+- server actions
+- database actions
+- collection actions
+
+The server actions are
+
+`create user`: allows to create a new user.
+
+`update user`: allows to change the access levels and details of an existing
+user.
+
+`drop user`: allows to delete an existing user.
+
+`create database`: allows to create a new database.
+
+`drop database`: allows to delete an existing database.
+
+The database actions are tied to a given database, and access
+levels must be set
+for each database individually. For a given database the actions are
+
+`create collection`: allows to create a new collection in the given database.
+
+`update collection`: allows to update properties of an existing collection.
+
+`drop collection`: allows to delete an existing collection.
+
+`create index`: allows to create an index for an existing collection in the
+given database.
+
+`drop index`: allows to delete an index of an existing collection in the given
+database.
+
+The collection actions are tied to a given collection of a given
+database, and access levels must be set for each collection individually.
+For a given collection the actions are
+
+`read document`: read a document of the given collection.
+
+`create document`: creates a new document in the given collection.
+
+`modify document`: modifies an existing document of the given collection,
+this can be an update or replace operation.
+
+`drop document`: deletes an existing document of the given collection.
+
+`truncate collection`: deletes all documents of a given collection.
+
+For the server category, a user needs the following access level
+to perform an action. The access levels are *Administrate* and
+*No access*:
+
+| action                | server level |
+|-----------------------|--------------|
+| create a database     | Administrate |
+| drop a database       | Administrate |
+| create a user         | Administrate |
+| update a user         | Administrate |
+| drop a user           | Administrate |
+
+For the database category, a user needs the following access
+level to the given database to perform an action. The access levels
+are *Administrate*, *Access* and *No access*.
```

Comment:
We will probably drop the file AuthorizationModel.md altogether. This is internal information that the user does not need. For the "grantDatabase" call I mention that the mapping is "rw" -> "Administrate", "Access" -> "ro" and "none" -> "No Access" and that we leave it at that for backward compatibilitiy. Where else should we explain this?